lib/ansible/modules/windows/win_dotnet_ngen.py

```diff
@@ -25,9 +25,9 @@
     - There are in fact two scheduled tasks for ngen but they have no triggers so aren't a problem.
     - There's no way to test if they've been completed.
     - The stdout is quite likely to be several megabytes.
+options: {}
```

Comment:
Please revert.

Reply:
Should this not be the default if it does not exist ? An empty dictionary.

Reply:
Fixed.

Reply:
Ah, now I see that you object it being relocated?

Reply:
Yes, especially as there aren't any other changed being made to this file